packages/@sanity/server/src/buildStaticFiles.ts

```diff
@@ -44,12 +44,12 @@ export async function buildStaticFiles(
   } = options
 
   debug('Writing Sanity runtime files')
-  await writeSanityRuntime({cwd, reactStrictMode: false, watch: false})
+  await writeSanityRuntime({cwd, reactStrictMode: false, watch: false, basePath})
 
   debug('Resolving vite config')
   let viteConfig = await getViteConfig({
     cwd,
-    basePath,
+    basePath: ensureTrailingSlash(basePath || '/'),
```

Comment:
We should now make `basePath` a require option for `getViteConfig` so we are not maintaing multiple default values.